bot/exts/info/pep.py

```diff
@@ -16,7 +16,7 @@
 
 ICON_URL = "https://www.python.org/static/opengraph-icon-200x200.png"
 BASE_PEP_URL = "http://www.python.org/dev/peps/pep-"
-PEPS_LISTING_API_URL = "https://api.github.com/repos/python/peps/contents?ref=master"
+PEPS_LISTING_API_URL = "https://api.github.com/repos/python/peps/contents?ref=main"
```

Comment:
why do we define the ref at all? it isn't required, and the api defaults to the default branch if no branch is provided.